src/python/espressomd/scafacos.pyx

```diff
@@ -45,13 +45,13 @@ IF SCAFACOS == 1:
             self._set_params_in_es_core()
 
         def valid_keys(self):
-            tmp = ["method_name", "method_params", "prefactor"]
+            tmp = self.required_keys().copy()
```

Comment:
Couldn't you do `return self.required_keys().copy().add("check_neutrality")`?

Reply:
I mean, wrapped in an `if` of course

Reply:
I just tried it, but the `add()` method of `set` returns None.

Reply:
and we also can't do `return self.required_keys().copy() + {"check_neutrality"}`

Reply:
Not so fluent :-)